api/webhook/taskcluster.go

```diff
@@ -81,24 +83,20 @@ func tcWebhookHandler(w http.ResponseWriter, r *http.Request) {
 
 // https://developer.github.com/v3/activity/events/types/#statusevent
 type statusEventPayload struct {
-	SHA       string      `json:"sha"`
-	State     string      `json:"state"`
-	Context   string      `json:"context"`
-	TargetURL string      `json:"target_url"`
-	Branches  branchInfos `json:"branches"`
+	github.StatusEvent
```

Comment:
Instead of embedding a single field, let's make `statusEventPayload` an alias of `github.StatusEvent`.

Reply:
Done.